### ISSUE TITLE
function-color-relative clashes with CSS saturate filter

### ISSUE THREAD

**Issue by delucis** [kristerkari/stylelint-scss#462]
With `function-color-relative` enabled, the following will warn even though it’s not a colour function:

```css
filter: saturate(50%);
```

`filter` values can only include colours within the `drop-shadow` filter:

```scss
filter: drop-shadow(0 0 5px saturate(red, 50%));
```

So maybe it’s possible to only apply `function-color-relative` inside `drop-shadow` for `filter`?

MDN Syntax Reference for `filter`: https://developer.mozilla.org/en-US/docs/Web/CSS/filter#Syntax

**Comment by kristerkari** [comment]
Yeah, the rule should ignore the `filter` property.

**Comment by kristerkari** [comment]
Fix released in v3.17.1
https://github.com/kristerkari/stylelint-scss/releases/tag/3.17.1

**Comment by delucis** [comment]
Thank you! 🎉
